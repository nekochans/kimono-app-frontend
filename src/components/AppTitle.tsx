import React from 'react';

type Props = {
  text: string;
};

const AppTitle: React.FC<Props> = ({ text }: Props) => (
  <>
    <h1>{text}</h1>
    <style jsx>{`
      h1 {
        color: red;
        font-size: 50px;
      }
    `}</style>
  </>
);

export default AppTitle;
