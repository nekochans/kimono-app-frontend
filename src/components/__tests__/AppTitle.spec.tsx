import React from 'react';
import renderer from 'react-test-renderer';
import AppTitle from '../AppTitle';
import { testProps } from '../AppTitle.stories';

test('AppTitle', () => {
  const component = renderer.create(<AppTitle {...testProps} />);
  const tree = component.toJSON();

  expect(tree).toMatchSnapshot();
});
