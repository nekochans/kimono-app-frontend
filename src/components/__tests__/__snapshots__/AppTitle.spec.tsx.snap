// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`AppTitle 1`] = `
Array [
  <h1>
    🐱KimonoApp🐱
  </h1>,
  <style
    jsx={true}
  >
    
      h1 {
        color: red;
        font-size: 50px;
      }
    
  </style>,
]
`;
